addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteForward.js

```diff
@@ -106,7 +108,15 @@ HTMLElement.prototype.oDeleteForward = function (offset) {
     }
 
     if (firstLeafNode && (isFontAwesome(firstLeafNode) || isNotEditableNode(firstLeafNode))) {
+        const nodeIsEditorTab = isEditorTab(firstLeafNode);
```

Comment:
Why create a variable for this ? It is only used once and the name does not add more information than the function call itself.